setup.py

```diff
@@ -71,9 +71,16 @@
     "test": ["parameterized", "pytest", "pytest-xdist", "accelerate", "peft>=0.4.0", "diffusers>=0.18.0"],
     "peft": ["peft>=0.4.0"],
     "diffusers": ["diffusers>=0.18.0"],
-    "deepspeed": ["deepspeed>=0.9.5"],
-    "dev": ["parameterized", "pytest", "pytest-xdist", "pre-commit", "peft>=0.4.0", "diffusers>=0.18.0"],
-    "benchmark": ["wandb", "ghapi", "openrlbenchmark==0.2.1a5", "requests", "deepspeed"],
+    "dev": [
+        "parameterized",
+        "pytest",
+        "pytest-xdist",
+        "pre-commit",
+        "peft>=0.4.0",
+        "diffusers>=0.18.0",
+        "deepspeed>=0.9.5",
+    ],
+    "benchmark": ["wandb", "ghapi", "openrlbenchmark==0.2.1a5", "requests", "deepspeed", "bitsandbytes>=0.41.0"],
```

Comment:
i think a small misunderstanding :) what i meant with my previous comment is that we can have a quantization and a deepspeed recipe but the dev recipe is ideally always the union of all recipes. does that make sense?

Reply:
Ah my bad. Just updated.